We.Sell.Bread.API.Unit.Tests/Tests/Services/ProductServiceTests.cs

```diff
@@ -19,7 +19,46 @@ public void GivenDataExistWhenRetrievingProductReturnTypeMustBeListOfProductDeta
 
             product.Should().NotBeNull();
             product.Should().BeOfType(typeof(List<ProductDetailsDto>));
-           
+
+        }
+
+        [Fact]
+        public void GivenIdShouldExistAndReturnTypeObjectProductDetailsDto()
+        {
+            ProductDetailsDto? product = _productService.GetProduct(new Guid("411fbad3-925e-4044-9269-f962641f9277"));
+            string realName = "Pumpernickel Bread";
+            decimal realPrice = (decimal)20.00;
+            string realDescription = "A dense, dark bread made from coarsely ground whole rye grains";
+            int realStock = 7;
+
+            product.Should().NotBeNull();
+            product.Should().BeOfType(typeof(ProductDetailsDto));
+            Assert.Equal(realName, product.ProductName);
+            Assert.Equal(realPrice, product.Price);
```

Comment:
Here we have made conscious decision in the project to use fluent assertions as the standard for our automated test cases, as it gives us more functionality than the xunit assertions, see line 34/35. can you adapt your assertions to follow that standard e.g product.Should().Be(realName);